src/controllers/TalksController.php

```diff
@@ -124,9 +124,14 @@ public function postAction($request, $db)
                         $comment    = $comment_mapper->getCommentById($new_id);
                         $speakers   = $talk_mapper->getSpeakerEmailsByTalkId($talk_id);
                         $recipients = array();
+                        $commentAuthorEmail = $comment[0]['email'];
```

Comment:
without looking at the code around it theres no way to know what  `$comment[0]['email']` is. The variable makes that clear (and readable without having to lookup the meaning)
Happy to change if required